browser/resources/settings/brave_default_extensions_page/brave_default_extensions_page.js

```diff
@@ -14,15 +14,19 @@ Polymer({
 
   /** @private {?settings.BraveDefaultExtensionsBrowserProxy} */
   browserProxy_: null,
+  hideRestartToast: false,
 
   /** @override */
   created: function() {
     this.browserProxy_ = settings.BraveDefaultExtensionsBrowserProxyImpl.getInstance();
+    this.browserProxy_.showRelaunchToast().then(show => {
+      this.hideRestartToast = !show;
+      this.$.needsRestart.hidden = this.hideRestartToast;
```

Comment:
This may be a micro-optimization, so consider it a nit: I'd feel more comfortable if this didn't require a DOM change on 99.9% of all Settings page loads (i.e. page loads which don't have the toast showing).
This can be achieved by either:
- Preferred: Wrap the element in an `<if` which tests `hideRestartToast` (though positive variable name is better, e.g. `showRestartToast`
- Similar pattern to what you have: Create a css class called something like `pageNeedsRestart`, and only add this class if the variable comes back true.